File: nevec_rs/src/err/write/color.rs
#[derive(Clone, Copy)]
pub enum Color {
    Red,
    Green,
    Blue,
    Yellow,
    Gray,
    Reset,
}

impl Color {
    pub fn code(&self) -> &str {
        match self {
            Color::Red => "\x1b[31m",
            Color::Green => "\x1b[32m",
            Color::Blue => "\x1b[34m",
            Color::Yellow => "\x1b[33m",
            Color::Gray => "\x1b[2m",
            Color::Reset => "\x1b[0m",
        }
    }
}

File: src/err/write/Color.kt
package err.write

/**
 * A limited set of colors and their codes for [err.msg.Msg]s.
 */
enum class Color(val code: String) {
    RED("\u001b[31m"),
    GREEN("\u001b[32m"),
    YELLOW("\u001b[33m"),
    BLUE("\u001b[34m"),
    GRAY("\u001b[2m"),
    NONE("\u001b[0m");
}